### ISSUE TITLE
How to plug a plugin in my own page?

### ISSUE THREAD

**Issue by paulserizay** [jacomyal/sigma.js#517]
Hi, I'm totally new at coding and i have some trouble using the dragNodes plugin. I can make the plugin work by itself with the generated random graph, but i am struggling to make it work with my own graph.

I don't understand what to put in the html. I tried to put only that:

<script src="../plugins/sigma.plugins.dragNodes/sigma.plugins.dragNodes.js"></script>
sigma.plugins.dragNodes(s, s.renderers[0]);

and it doesn't make my points move. I think i have to change datas of "s" and "s.renderers[0]" but i can't find where and i don't know how to manage to do it...

Basically, I would love if anyone could give me some explanations about how to plug a plugin in my page?
If you coul'd help me, I'm lost and that would be awsome! Thank you a lot!
